packages/rules/src/alpha.ts

```diff
@@ -8,7 +8,7 @@ import { FormKitValidationRule } from '@formkit/validation'
  */
 const alpha: FormKitValidationRule = function ({ value }, set = 'default') {
   const sets = {
-    default: /^[a-zA-ZÀ-ÖØ-öø-ÿĄąĆćČčĎďĘęĚěŁłŃńŇňŘřŚśŠšŤťŮůŹźŻŽžż]+$/,
+    default: /^[\p{Lu}\p{L}]+$/,
```

Comment:
Don’t forget to add the unicode flag on these 👍 (switching to Unicode is a good suggestion by the way)
```suggestion
    default: /^[\p{Lu}\p{L}]+$/u,
```